Code/GraphMol/MolDraw2D/test1.cpp

```diff
@@ -47,7 +47,7 @@ using std::regex_search;
 
 // if 0, turns off a lot of the FREETYPE TEST_ASSERT checks
 // because sometimes you want to look at all the pictures first.
-#define DO_TEST_ASSERT 1
+#define DO_TEST_ASSERT 0
```

Comment:
should remember to turn this back on before merging the PR

Reply:
Once everything else is accepted, I'll go through all the tests and make sure they're working correctly.